src/main/resources/com/google/api/codegen/nodejs/method_sample.snip

```diff
@@ -216,24 +211,24 @@
 @end
 
 @private methodCallSampleCodeWithReturnValue(apiMethod, additionalCallback)
-  @if apiMethod.isLongRunningOperation
-    # TODO(landrito) figure out how to remove initcode repetition for method samples.
-    {@methodCallSampleCodeLongrunningPromise(apiMethod, additionalCallback)}
+  {@initCode(apiMethod)}
+  {@methodCallSampleCodePrefix(apiMethod)}
+    .then(responses => {
+      var response = responses[0];
+      @if additionalCallback
+        console.log(response);
+      @else
+        // doThingsWith(response)
+      @end
+    })
+    {@promiseCallbacks(additionalCallback)}
+@end
 
-    {@methodCallSampleCodeLongrunningEventEmitter(apiMethod, additionalCallback)}
-  @else
-    {@initCode(apiMethod)}
-    {@methodCallSampleCodePrefix(apiMethod)}
-      .then(responses => {
-        var response = responses[0];
-        @if additionalCallback
-          console.log(response);
-        @else
-          // doThingsWith(response)
-        @end
-      })
-      {@promiseCallbacks(additionalCallback)}
-  @end
+@private operationSampleCode(apiMethod, additionalCallback)
```

Comment:
What do you think about a more descriptive name, either `longRunningOperationSampleCode` or `lroSampleCode`?

Reply:
Done